lix/packages/sdk/src/schema.ts

```diff
@@ -67,6 +74,8 @@ export type Change = {
 	value?: Record<string, any> & {
 		id: string
 	} // JSONB
+
+	conflict?: any[] | null
```

Comment:
I recommend using undefined for types (which you accidentally did?). Using `null` opens 3 possible states for a value `value | undefined | null` where `null` might or might not convey different meaning than `undefined`. 

Also unsure how conflict could be an array. If conflict points to a conflict_id, it can only be a string. If conflict is an array of changes, I would be confused. 

```suggestion
	conflict?: any
```

Reply:
@samuelstroschein  i removed the conflict until i the pr for merging. but still unsure about how to type for sql. in sql null is used for empty cells and this is also how kysely works.